packages/testsuite/cypress.config.ts

```diff
@@ -30,47 +29,134 @@ export default defineConfig({
               ])
               .withWaitStrategy(Wait.forLogMessage(new RegExp(".*(WildFly Full.*|JBoss EAP.*)started in.*")))
               .withStartupTimeout(333000)
-              .withCommand(["-c", configuration || "standalone-insecure.xml"] as string[])
+              .withCommand(["-c", configuration || "standalone-insecure.xml"] as string[]);
+            if (useNetworkHostMode === true) {
+              console.log("host mode");
+              wildfly.withNetworkMode("host");
+            } else {
+              console.log("network mode");
+              wildfly
+                .withNetworkMode(config.env.NETWORK_NAME as string)
+                .withNetworkAliases("wildfly")
+                .withExposedPorts(9990);
+            }
+            wildfly
               .start()
-              .then((wildflyContainer) => {
+              .then(async (wildflyContainer) => {
                 startedContainers.set(name as string, wildflyContainer);
-                const managementApi = `http://localhost:${wildflyContainer.getMappedPort(9990)}/management`;
-                return axios
-                  .post(managementApi, {
-                    operation: "list-add",
-                    address: ["core-service", "management", "management-interface", "http-interface"],
-                    name: "allowed-origins",
-                    value: `http://localhost:${config.env.HAL_CONTAINER_PORT as string}`,
-                  })
-                  .then(() => {
-                    return axios.post(managementApi, {
-                      operation: "reload",
+                if (useNetworkHostMode === false) {
```

Comment:
This seems dangerous, what if it's not set? I suggest to compare against `true` which is more explicit.

Reply:
compare against `true` is used